src/main/kotlin/com/example/mykku/image/NoOpImageUploadService.kt

```
@@ -34,4 +34,7 @@ class NoOpImageUploadService : ImageUploadService {
     ): EntityImagesUploadResult {
         throw ImageException.imageUploadServiceUnavailable()
     }
+
+    override fun delete(url: String) {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)